g_python/hunitytools.py

```diff
@@ -1,14 +1,12 @@
 import threading
-import time
 
 from .gextension import Extension
 from .hmessage import HMessage, Direction
-from .hpacket import HPacket
-from .hunityparsers import HUnityEntity
+from .hunityparsers import HUnityEntity, HUnityStatus
 
 
 class UnityRoomUsers:
-    def __init__(self, ext: Extension, users_in_room=28, get_guest_room=385, user_logged_out=29):
+    def __init__(self, ext: Extension, users_in_room=28, get_guest_room=385, user_logged_out=29, status=34):
```

Comment:
could you replace the header IDs with the packet names (does that work in g-python for unity? it should)

Reply:
I'll try if it works and then change it!

Reply:
Does not work unfortunately

Reply:
You're aware thar the names changed right?

Reply:
yes, I took the names from messages.json
https://pastebin.com/mMTTkxWn

This is the version I currently have locally installed

Reply:
do packet names work on the latest commit for you?

Reply:
merging for now but making issue for this